modules/aws/elasticacheRedis/outputs.tf

```diff
@@ -1,3 +1,45 @@
-output "redis_host" {
-  value = aws_elasticache_cluster.my_cluster.cache_nodes[0].address
-}
\ No newline at end of file
+
+// output "public_subnet_cidrs" {
+//   value       = module.subnets.public_subnet_cidrs
+//   description = "Public subnet CIDRs"
+// }
+
+// output "private_subnet_cidrs" {
+//   value       = module.subnets.private_subnet_cidrs
+//   description = "Private subnet CIDRs"
+// }
+
+// output "vpc_cidr" {
+//   value       = module.vpc.vpc_cidr_block
+//   description = "VPC CIDR"
+// }
+
+output "cluster_id" {
+  value       = module.redis.id
+  description = "Redis cluster ID"
+}
+
+output "cluster_endpoint" {
+  value       = module.redis.endpoint
+  description = "Redis primary endpoint"
+}
+
+output "cluster_host" {
+  value       = module.redis.host
+  description = "Redis hostname"
+}
+
+// output "cluster_security_group_id" {
```

Comment:
do you want to delete the commented out parts?

Reply:
removed most of them that aren't relevant